### ISSUE TITLE
Page explaining how to get a workshop!

### ISSUE THREAD

**Issue by thursdayb** [safetyfirstpdx/safetyfirstpdx.github.io#4]
Even if it's just an email address with the words "Contact us to schedule a workshop!"

**Comment by gabelula** [comment]
We already did this!

**Comment by thursdayb** [comment]
Yeah, this is a marketing page for businesses or organizations that want a private workshop

**Comment by thursdayb** [comment]
This is a draft of the text: 

Training Program > Onsite Training
 
Need training for your company or organization? SafetyFirst PDX provides private onsite training for companies, organizations, and communities. Please contact us [by doing this thing] to check availability and pricing.

@aeschright @gabelula How do we want folks to contact us for arranging private training? Email? Contact form?

**Comment by gabelula** [comment]
Email? training@stumptownsyndicate.org ?

**Comment by gabelula** [comment]
Also, shoud this text be in the same page (http://safetyfirstpdx.org/training/code_of_conduct/) where we talk about the workshop. Maybe right before "Training materials".

**Comment by thursdayb** [comment]
Maybe safetyfirstpdx@stumptownsyndicate.org? In case Stumptown Syndicate ever decides to offer other kinds of training?
